packages/epo-react-lib/.storybook/decorators.tsx

```diff
@@ -8,7 +8,7 @@ const withTheme: DecoratorFn = (StoryFn) => {
   return (
     <>
       <GlobalStyle />
-      <StoryFn />
+      {StoryFn()}
```

Comment:
needed to change to function syntax or else this decorator broke using templates